server/game/cards/12-KotI/GunthorSonOfGurn.js

```diff
@@ -0,0 +1,35 @@
+const DrawCard = require('../../drawcard.js');
+
+class GunthorSonOfGurn extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                afterChallenge: event => (
+                    event.challenge.attackingPlayer === this.controller &&
+                    event.challenge.winner === this.controller &&
+                    this.isAttacking()
+                )
+            },
+            handler: context => {
+                this.game.promptForSelect(context.opponent, {
```

Comment:
I'm not sure `context.opponent` will automatically be filled in here (though maybe it should). I think it's exclusive to cards that use `chooseOpponent`. Use `context.event.challenge.loser` instead.